### DISCUSSION TITLE
memory-cache missing exportJson, importJson type declarations

### DISCUSSION THREAD

**Discussion by k4rli** [DefinitelyTyped/DefinitelyTyped#58547] [Issues with a @types package] [answered]
Type definitions for ptarjan/node-cache on Github or memory-cache on npm don't match the library.

[Type declarations here](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/memory-cache/index.d.ts) are missing [`exportJson`](https://github.com/ptarjan/node-cache#exportjson--function) and [`importJson`](https://github.com/ptarjan/node-cache#importjson--functionjson-string-options--skipduplicates-boolean-) functions.

[a comment by an automated account is not shown]

**Comment by MysteryBlokHed** [comment, marked as answer]
I just opened #58548
